test/framework/management/kind/mgmt.go

```diff
@@ -182,15 +214,8 @@ func (c *Cluster) Teardown(ctx context.Context) {
 	if c == nil {
 		return
 	}
-	deleteCmd := exec.NewCommand(
-		exec.WithCommand("kind"),
-		exec.WithArgs("delete", "cluster", "--name", c.Name),
-	)
-	stdout, stderr, err := deleteCmd.Run(ctx)
-	if err != nil {
+	if err := cluster.NewProvider(cluster.ProviderWithLogger(cmd.NewLogger())).Delete(c.Name, ""); err != nil {
```

Comment:
If we put the explicit `Kubeconfig` back in this will need updating as I believe the second argument to `Delete` should be the Kubeconfig path